interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +126,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$edit && ($_GET['group']==="true" || $_GET['group']==true ))?' disabled=true; ':'';?>
```

Comment:
Not saying you should write  "!empty($_GET['group'])". What I'm saying is your code is equivalent to that. Specifically,  "$_GET['group']" when converted to boolean is true when non-empty and nonzero.  See:
http://php.net/manual/en/language.types.boolean.php